src/main/scala/viper/gobra/frontend/ParseTreeTranslator.scala

```diff
@@ -2396,6 +2490,12 @@ class ParseTreeTranslator(pom: PositionManager, source: Source, specOnly : Boole
       case c if cond(c) => visitNode(c)
     }.asInstanceOf[Vector[P]]
   }
+
+  def visitListNodeIf[P <: PNode, C <: ParserRuleContext](ctx: java.util.List[C], cond: (C => Boolean), select : (C => java.util.List[_ <: ParserRuleContext])): Vector[Vector[P]] = {
+    ctx.asScala.toVector.collect {
+      case c if cond(c) => visitListNode(select(c))
+    }.asInstanceOf[Vector[Vector[P]]]
+  }
```

Comment:
What is this doing? Add comments or remove it if it is not necessary

Reply:
not needed anymore